src/features/RentWorks/components/AddProperty/AddProperty.jsx

```diff
@@ -159,6 +160,53 @@ export default function AddProperty({
           />
         </Stack>
 
+        {/* Initial Late Fee and Daily Late Fee */}
+        <Stack direction="row" spacing={2}>
+          <TextFieldWithLabel
```

Comment:
These late fees and charges can be customized according to the renter. We need to fix that. Also we need to display a list of charges for the client during checkout phase. This must also be listed there if possible. ( Requested by the client )